sdk/deploy.js

```diff
@@ -55,6 +55,7 @@ const deployCode = async function deployCode(deployURLBase, apiKey, tarPath) {
  * @return {object} - response of tag operation
  */
 const deployConf = async function deployConf(deployURLBase, apiKey, funcName, funcConf) {
+  const interpolRegex = new RegExp(/^\$[a-zA-Z_]{1,}[a-zA-Z0-9_]{0,}/);
```

Comment:
We should be more clear about the supported syntax for interpolation.
`$THIS` - sure
`${WHAT_ABOUT_THIS}`?
What would a user expect? How is this documented (should probably be part of this PR).

Reply:
I worry about this home-grown implementation of envar expansion. What about the various edge cases? How do you escape `$`? I also think `regex` is a "now you have two problems" type of solution to this problem.

Maybe try to find a NPM package that handles this? e.g. this impl seems to be a bit more robust: https://github.com/danday74/envsub/blob/master/js/envsub-parser.js
If you can't find a good one and you decide to implement it yourself, you should probably publish it to NPM publicly. Useful.

Alternatively, use a proper template engine, like handlebars. While the syntax won't be the same as the familiar SH syntax for interpolation, it's still a widely known syntax. And of course the benefit is that we don't roll our own parser.